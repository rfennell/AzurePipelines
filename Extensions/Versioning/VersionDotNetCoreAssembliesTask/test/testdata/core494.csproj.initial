<Project Sdk="Microsoft.NET.Sdk">
  <PropertyGroup>
    <TargetFramework>net472</TargetFramework>
    <AssemblyVersion>1.2.3.4</AssemblyVersion>
  </PropertyGroup>
  <PropertyGroup>
    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
  </PropertyGroup>
</Project>