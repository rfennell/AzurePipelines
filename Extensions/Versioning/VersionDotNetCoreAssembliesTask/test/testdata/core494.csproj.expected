<Project Sdk="Microsoft.NET.Sdk">
  <PropertyGroup>
    <TargetFramework>net472</TargetFramework>
    <AssemblyVersion>9.9.9.9</AssemblyVersion>
  </PropertyGroup>
  <PropertyGroup>
    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
  </PropertyGroup>
</Project>